app/webpacker/components/RegistrationsV2/Registrations/RegistrationList.jsx

```diff
@@ -94,8 +94,10 @@ export default function RegistrationList({ competitionInfo }) {
   const { sortColumn, sortDirection } = state;
   const changeSortColumn = (name) => dispatch({ type: 'CHANGE_SORT', sortColumn: name });
 
+  const blindFoldedEvents = ['333mbf', '333bf', '444bf', '555bf'];
+
   const [psychSheetEvent, setPsychSheetEvent] = useState();
-  const [psychSheetSortBy, setPsychSheetSortBy] = useState('single');
+  const psychSheetSortBy = useMemo(() => (blindFoldedEvents.includes(psychSheetEvent) ? 'single' : 'average'), [psychSheetEvent]);
```

Comment:
Using my suggestion from above, you can compute this as `events.byId[psychSheetEvent].recommendedFormat().sortBy`